src/pages/Dashboard/Leaderboard.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { Box, Flex, VStack, Heading, Text, Table, Tr, Td, Th, Thead, Tbody } from '@chakra-ui/react';
+import { useSelector } from 'react-redux';
+
+const GET_GLOBAL_LEADERBOARD = gql`
+  query getGlobalLeaderboard {
+    getGlobalLeaderboard {
+      id
+      name
+      organization
+      itemsSold
+    }
+  }
+`;
+
+const composeTableHeader = (header, i) => {
+  return (
+    <Th bgColor="white" key={i}>
+      <Heading textTransform="uppercase" as="p" size="subtitle" color="black" opacity="0.5" mb="8px">
+        {header}
+      </Heading>
+    </Th>
+  );
+};
+
+const TableRow = props => {
+  const { bgColor, borderBottomWidth, rank, teamName, affiliation, itemsSold } = props;
+  return (
+    <Tr key={rank} borderBottomWidth={borderBottomWidth} borderColor="white" bg={bgColor} height="88px">
+      <Td>
+        {(rank === 1 && (
+          <Text fontSize="40px">
+            <span role="img" aria-label="gold medal">
+              🥇
+            </span>
+          </Text>
+        )) ||
+          (rank === 2 && (
+            <Text fontSize="40px">
+              <span role="img" aria-label="gold medal">
+                🥈
+              </span>
+            </Text>
+          )) ||
+          (rank === 3 && (
+            <Text fontSize="40px">
+              <span role="img" aria-label="gold medal">
+                🥉
+              </span>
+            </Text>
+          )) || (
+            <Text opacity="0.5" paddingLeft="6px">
+              {rank}
+            </Text>
+          )}
```

Comment:
I think we can do better than this! 
```suggestion
 {rank <= 3 ? (
          <Text fontSize="40px">
            <span role="img" aria-label={`${rank === 1 ? 'gold' : rank === 2 ? 'silver' : 'bronze'} medal`}>
              {rank === 1 ? '🥇' : rank === 2 ? '🥈' : '🥉'}
            </span>
          </Text>
        ) : (
          <Text opacity="0.5" paddingLeft="6px">
            {rank}
          </Text>
        )}
```

Reply:
fixed